.github/workflows/issue-triage.yml

```diff
@@ -81,4 +81,32 @@ jobs:
             [`repl`](https://babeljs.io/repl/), or if the `repl` is
             insufficient, a new and minimal repository with instructions on
             how to build/replicate the issue.
-        
+
+  npm_cache:
+    name: Npm Cache
+    runs-on: ubuntu-latest
+    steps:
+      - name: Npm Cache Comment
+        uses: babel/actions/create-comment@v2
+        if: |
+          github.event.action == 'labeled' &&
+          github.event.label.name == 'i: npm cache'
+        with:
+          token: ${{ secrets.BOT_TOKEN }}
+          issue: ${{ github.event.issue.number }}
+          comment: >
+            Hi @${{ github.event.issue.user.login }}!
+
+            This seems to be an npm related issue: you can verify that the package is available using the npm website.
```

Comment:
> an npm related issue

Wonder if we should call it an "unavailable package issue" instead?

Reply:
```suggestion
            This seems to be an issue caused by a package that you are not able to download: you can verify that the package is available using the npm website.
```